### ISSUE TITLE
No command lines are shown when i click the verify button in Arduino IDE

### ISSUE THREAD

**Issue by Algorithmss** [me-no-dev/RasPiArduino#46]

**Comment by me-no-dev** [comment]
You need to enable verbose compilation in ArduinoIDE settings.
![screen shot 2016-06-19 at 11 08 56](https://cloud.githubusercontent.com/assets/12663778/16176131/5ac4ff82-360e-11e6-83f3-383621474b5e.png)

**Comment by Algorithmss** [comment]
And also can you help me with this error? 

exec: "C:\\Program Files (x86)\\Arduino\\hardware\\RaspberryPi\\piduino/tools/arm-linux-gnueabihf/bin/arm-linux-gnueabihf-g++": file does not exist

**Comment by Algorithmss** [comment]
I got it to work once then i copied the bin file into my raspberry pi but it says that i cannot run it, it shows me " command not found " error

**Comment by me-no-dev** [comment]
do you have the toolchain installed on your computer? If you do and you build, you can export the binary to the sketch folder, then copy to the py, make it executable (chmod +x your_file.bin), then run it like above (it has to be executable for that command to work)

**Comment by Algorithmss** [comment]
Okay thanks, let me try that method out :)

**Comment by Algorithmss** [comment]
Thanks it works... :D but it just keep on looping and looping and it goes on and on and i can't make it stop

**Comment by me-no-dev** [comment]
that is how an arduino sketch works :) Ctrl+C to stop it

**Comment by me-no-dev** [comment]
if you follow the readme and do the modifications on the PI, you will be able to upload directly from the PC and the service will take care of killing the previous sketch

**Comment by Algorithmss** [comment]
Yup i want to do that too, i will try it out. And also, i can just write the code down in Arduino language right? Or is there a difference?

**Comment by me-no-dev** [comment]
no difference :) it is arduino framework. so Wire, SPI and all is there for you to use. Most sketches require usually only pin change for them to work on the Pi

**Comment by Algorithmss** [comment]
Oh okayy sounds great, Thanks for the help Mr. I am new to all this and i want to try out projects :)

**Comment by Algorithmss** [comment]
Oh and is there any chance i can use this with Blynk app? If yess tell me how :) 
Thanks x)

**Comment by me-no-dev** [comment]
what is Blynk app?

**Comment by Algorithmss** [comment]
It is basically like a remote app that control your devices, from the app. http://www.blynk.cc/   here's the link to the app website. I am bad at explaining so wanna check it out? it is really cool

**Comment by Algorithmss** [comment]
Ohh also, when i installed this, does it still function as a normal Raspberry Pi? Like can i put others code in? or like add some git code inside?

**Comment by me-no-dev** [comment]
yes, it's just another program that runs on your PI :) you can still use it as a linux box or whatever you want. Just keep in mind to not use the GPIO pins for anything else while the sketch is running, because it controls the whole pinout
